documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,78 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. You can also use the HDMI and DSI display outputs at the same time.
```

Comment:
The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is a 7-inch LCD display that can be connected to a Raspberry Pi using the DSI connector. While the panel is connected, both it, and the normal HDMI display output can be used at the same time.

Reply:
Thank you, my brain couldn't figure that one out yesterday. Modified it a bit to be more active voice though.